draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.spec.tsx

```diff
@@ -1,42 +1,82 @@
 import React from "react"
-import { render } from "@testing-library/react"
-import * as AppearanceAnim from "./AppearanceAnim"
+import { render, screen, waitFor } from "@testing-library/react"
+import { Button } from "@kaizen/button"
 import { Tooltip } from "./index"
-import "@testing-library/jest-dom"
-jest.mock("./AppearanceAnim")
-const AnimationProvider = AppearanceAnim.AnimationProvider as jest.Mock
 
 describe("<Tooltip />", () => {
-  beforeEach(() => {
-    AnimationProvider.mockReturnValue(<div />)
-  })
-  describe("When no animationDuration prop is given", () => {
-    it("animationDuration prop is passed to AnimationProvider as undefined", () => {
+  describe("Accessible descriptions", () => {
+    it("will render a button will have an accessible description", async () => {
       render(
-        <Tooltip text="Example">
-          <div />
+        <Tooltip
+          text="Tooltip popup description for button"
+          display="inline"
+          isInitiallyVisible
+          position="below"
+        >
+          <Button label="More info" />
         </Tooltip>
       )
-      expect(AnimationProvider).toHaveBeenCalledWith(
-        expect.objectContaining({
-          animationDuration: undefined,
-        }),
-        {}
+
+      await waitFor(() => {
+        expect(
+          screen.getByRole("button", { name: "More info" })
+        ).toHaveAccessibleDescription("Tooltip popup description for button")
+      })
+    })
+
+    it("will render a button without an aria-describedby if the tooltip is not active", async () => {
+      render(
+        <Tooltip
+          text="Tooltip popup description for button"
+          display="inline"
+          position="below"
+        >
+          <Button label="More info" />
+        </Tooltip>
       )
+
+      await waitFor(() => {
+        expect(screen.queryByRole("tooltip")).toBe(null)
+        expect(
+          screen.getByRole("button", { name: "More info" })
+        ).not.toHaveAttribute("aria-describedby")
+      })
     })
-  })
-  describe("When animationDuration prop is given", () => {
-    it("animationDuration prop is passed to AnimationProvider", () => {
+
+    // Non-semantic elements without roles should not have aria-description on them - they will not read to screen readers and will throw an error
+    it("will render non-semantic elements without accessible descriptions", async () => {
       render(
-        <Tooltip text="Example" animationDuration={1000}>
-          <div />
+        <Tooltip
+          text="Tooltip popup description for div"
+          display="inline"
+          isInitiallyVisible
+          position="below"
+        >
+          <div>Non somantic element</div>
```

Comment:
Or was it on purpose 😂 
```suggestion
          <div>Non semantic element</div>
```

Reply:
Nope 😆